tests/view/conftest.py

```diff
@@ -4,15 +4,14 @@
     matplotlib.use('Agg')  # noqa
 
 
-from contextlib import contextmanager
-
 from neurom.view import common
 common._get_plt()
 
 from neurom.view.common import plt
+import pytest
 
 
-@contextmanager
+@pytest.yield_fixture
```

Comment:
With pytest >= 3.0 this should be just `@pytest.fixture`, because from https://docs.pytest.org/en/stable/yieldfixture.html:

> Marking functions as yield_fixture is still supported, but deprecated and should not be used in new code.

Reply:
Thank you! I was unaware.